### ISSUE TITLE
Validate custom attribute no longer working?

### ISSUE THREAD

**Issue by Souldrinker** [aurelia/validation#227]
Does the validateCustomAttribute for aurelia-validation still work? Since the ensure decorators broke a while back after I upgraded to core-js 2.x I've been living without the validation awaiting a fix for the decorators, but with yesterdays release it seems like the ensure decorators are deprecated and won't be coming back, thus I changed my code to use the "normal" ensure chaining, but it seems like it still won't work.
I have my data in a separate class than the view model and has thus relied on the ```validate``` custom attribute to connect the property and the ensure. I tried in the latest skeleton too (which uses core-js 1.x), but I get the same behavior there, that is if I just add this to the constructor in welcome.js:
```
    this.person = person;
    this.validation = validation.on(this)
        .ensure('firstName')
        .isNotEmpty()
        .hasLengthBetween(3,10);
```
...add a ```validate.bind="validation"``` on the form in welcome.html and change the form-group to this:
```
    <div class="form-group">
        <label for="fn">First Name</label>
        <input type="text" value.bind="firstName" class="form-control" id="fn" placeholder="first name">
      </div>
```
...it works correctly.

But if I add a person.js like this:
```
export class Person{
    firstName = "John";
}
```
...and change the html to this (bind to person.firstName and add the validate custom attribute):
```
      <div class="form-group">
        <label for="fn">First Name (Person)</label>
        <input type="text" value.bind="person.firstName" class="form-control" id="fn" placeholder="first name" validate="firstName">
      </div>
```
...the validation doesn't seem to work.

Is it me doing something wrong or has something else broken? Because I'm pretty sure this worked a couple of weeks ago before I upgraded to aurelia-validation 0.6.2 & core-js 2.x, even though I at the time used ensure decorators rather than chaining ensures while setting up the validation.

**Comment by EisenbergEffect** [comment]
@PWKad Can you look into this? I believe the binding expression has to match the ensure method call expression at the moment. That may be what is wrong. Ultimately, that needs to be changed if that's the case.

**Comment by Souldrinker** [comment]
Sorry about this. It seems like our css guy did some changes while the ensure decorators where broken where he changed ```form-group``` to ```formgroup``` for some reason that he can't really remember now. And then when you deprecated the ensure decorators and I rewrote our code to use ensure chaining the validation didn't work since it didn't find any form-groups.

And when I tried in the skeleton it seems like I forgot to change the ```validation.on(this)``` to ```validation.on(this.person)```, so it was kind of a double failure from our side.

So, now when we were about to deploy soon to our first end users and I was about to rewrite the code to not use the seemingly broken aurelia-validation stuff we noticed this and things started working again when we renamed the formgroups :-)
